plot_app/configured_plots.py

```diff
@@ -352,19 +352,22 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
                 airspeed_validated = ulog.get_dataset('airspeed_validated')
                 data_plot.change_dataset('airspeed_validated')
                 if np.amax(airspeed_validated.data['airspeed_sensor_measurement_valid']) == 1:
-                    data_plot.add_graph(['equivalent_airspeed_m_s'], colors8[1:2],
-                                        ['Equivalent Airspeed'])
+                    data_plot.add_graph(['true_airspeed_m_s'], colors8[1:2],
+                                        ['True Airspeed'])
                 else:
-                    data_plot.add_graph(['equivalent_ground_minus_wind_m_s'], colors8[1:2],
-                                        ['Ground Minus Wind'])
+                    data_plot.add_graph(['true_ground_minus_wind_m_s'], colors8[1:2],
+                                        ['True Airspeed (estimated)'])
             else:
                 data_plot.change_dataset('airspeed')
                 data_plot.add_graph(['indicated_airspeed_m_s'], colors8[1:2],
                                     ['Indicated Airspeed'])
             data_plot.change_dataset('vehicle_gps_position')
             data_plot.add_graph(['vel_m_s'], colors8[2:3], ['Ground Speed (from GPS)'])
             data_plot.change_dataset('tecs_status')
-            data_plot.add_graph(['airspeed_sp'], colors8[3:4], ['Airspeed Setpoint'])
+            if 'true_airspeed_sp' in cur_dataset.data:
```

Comment:
Can you move this to https://github.com/PX4/flight_review/blob/8c51ecfaa0ce1a06154c512d7696a19c497ed71e/plot_app/configured_plots.py#L39 renaming the field if it's an old log? I'd like to keep compatibility code at a common place.

Reply:
@bkueng like I did in b15b5fdae18b2dcb5273dbffe0551e0b7a736f02 now?